backend/docker-compose.dev.yaml

```diff
@@ -1,8 +1,29 @@
 version: "3.8"
 
 services:
+  keycloak:
+    container_name: keycloak-aof
+    image: quay.io/keycloak/keycloak:15.1.0
+    volumes:
+      - ./keycloak:/tmp/io
+    environment:
+      - KEYCLOAK_USER=admin
+      - KEYCLOAK_PASSWORD=admin
+      - KEYCLOAK_IMPORT=/tmp/io/dev-realm.json
+      - KEYCLOAK_STATISTICS=all # enables the health endpoint under 9990
+    ports:
+      - 8080:8080
+      - 9990:9990
```

Comment:
This seems to be the wildfly admin console. Does this need to be exposed?

Reply:
Only because of the health endpoint. I wanted to avoid it but couldn't find an easy alternative. I have an open question with the keycloak community. If they reply/recommend something else I can refactor in the future.